tests/framework/Samplers/AdaptiveLimitSurfaceSearch/test_adaptive_sampler_1_d_limit_surface.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" ?>
+<Simulation>
+  <!-- TestInfo -->
+  <TestInfo>
+    <name>framework/Samplers/AdaptiveLimitSurfaceSearch.limit_surface_1_dimensional</name>
+    <author>alfoa</author>
+    <created>2018-11-29</created>
+    <classesTested>Samplers.LimitSurfaceSearch, Models.ExternalModel, Models.ROM</classesTested>
+    <description>
+        This  test is aimed to test the capability of RAVEN to employ a goal oriented sampling. It tests the
+        LimitSurfaceSearch algorithm using an external model as ``system code''.
+        In addition it ensures that a 1Dimensional Limit surface can be generated (1 uncertanty).
+    </description>
+    <revisions>
+        <revision author="alfoa" date="2018-11-29">New test for addressing issue #864</revision>
+    </revisions>
+  </TestInfo>
+
+  <RunInfo>
+    <WorkingDir>Adapt1D</WorkingDir>
+    <Sequence>adapt,adaptdump</Sequence>
+    <batchSize>1</batchSize>
+    <maxQueueSize>1</maxQueueSize>
+  </RunInfo>
+
+  <Steps>
+    <MultiRun name="adapt" pauseAtEnd="true">
+      <Input class="DataObjects" type="PointSet">dummy</Input>
+      <Model class="Models" type="ExternalModel">testFunction</Model>
+      <Sampler class="Samplers" type="LimitSurfaceSearch">adaptiveSearch</Sampler>
+      <SolutionExport class="DataObjects" type="PointSet">limitSurface</SolutionExport>
+      <Output class="DataObjects" type="PointSet">sampledPoints</Output>
+      <Output class="OutStreams" type="Print">limitSurfaceDump</Output>
```

Comment:
Output using OutStreams can be removed, since additional IOStep is used to print the data.